pkg/controller/controller.go

```diff
@@ -296,7 +296,7 @@ func (c *Controller) sync(domain string) error {
 func (c *Controller) zmsGetSignedDomains(domain string) (*zms.SignedDomains, bool, error) {
 	d := zms.DomainName(domain)
 	master := false
-	signedDomain, _, err := c.zmsClient.GetSignedDomains(d, "", "", &master, "")
+	signedDomain, _, err := c.zmsClient.GetSignedDomains(d, "", "", &master,&master, "")
```

Comment:
Same as this comment: https://github.com/AthenZ/k8s-athenz-syncer/pull/38/files#r1618011313 .

Reply:
Hi @bhaskaramVaranasi ,
Please find the attached screenshot of the function `GetSignedDomains()`.
The `conditions` field has been added as a parameter to the updated package. So to accomodate this, we are sending it as a boolean set to `false`.

![Screenshot 2024-06-03 at 10 17 01 AM](https://github.com/AthenZ/k8s-athenz-syncer/assets/65297307/43b2a19a-cf10-4785-9e86-266c303b4bff)